cmake/OmimOptions.cmake

```diff
@@ -0,0 +1,19 @@
+set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Build type")
+option(CMAKE_UNITY_BUILD "Use unity build" ON)
+set(CMAKE_UNITY_BUILD_BATCH_SIZE "50" CACHE STRING "Batch size for unity build")
```

Comment:
Why is it cached instead of being an option?

Reply:
Yes, options are only for bools

Reply:
And option doesn't work in this case (like for `NJOBS`) because it's a cmake's variable